### PR TITLE
Add MyCommand to discord bot

### PR DESCRIPTION
**Pull request by ludeeus**
image

### CODE REVIEW

File: services/bots/src/discord/commands/my.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      embeds: [
+        new EmbedBuilder({
+          title: redirectData.name,
+          description: redirectData.description,
+          url: redirectData.params
+            ? `https://my.home-assistant.io/create-link/?redirect=${redirectData.redirect}`
+            : `https://my.home-assistant.io/redirect/${redirectData.redirect}/`,
+          ...(redirectData.params
+            ? { footer: { text: 'This My link needs adjustments before usage' } }
+            : {}),
```
Comment (balloob) [inline services/bots/src/discord/commands/my.ts:94]: Feels a little ugly. Is this possible?
```suggestion
          footer: redirectData.params
            ? { footer: { text: 'This My link needs adjustments before usage' } }
            : undefined,
```

File: services/bots/src/discord/commands/my.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      return;
+    }
+
+    await interaction.reply({
+      embeds: [
+        new EmbedBuilder({
+          title: redirectData.name,
+          description: redirectData.description,
+          url: redirectData.params
+            ? `https://my.home-assistant.io/create-link/?redirect=${redirectData.redirect}`
```
Comment (balloob) [inline services/bots/src/discord/commands/my.ts:90]: Could we dynamically expand the accepted parameters like `/my integration hue` 

(I guess we can do that in a future PR)

**Review by balloob — APPROVED** [review]
Super cool!

File: services/bots/src/discord/commands/my.ts (hunk 2 above)
Comment (ludeeus) [reply services/bots/src/discord/commands/my.ts:90]: Thats not possible.
Maybe we can add some form of prompt, but not sure.
Will have a deeper look later (for another PR)

Comment (balloob) [reply services/bots/src/discord/commands/my.ts:90]: Can we hardcode them then? At least one for a specific integration would be very useful.

**Pull request merged**